frontend/src/App.vue

```diff
@@ -4,15 +4,17 @@
     <HeaderComponent />
     <SnackBar />
     <NavBar v-if="pageTitle && isAuthenticated && showNavBar" :title="pageTitle" />
-    <v-main fluid class="align-start">
+    <v-main fluid style="--v-layout-top: 64px;" class="align-start">
+      <div class="v-main__wrap">
       <v-app-bar
         v-if="bannerColor !== ''"
-        style="color: white"
+        :style="{ height: '20px', color: 'white', position: 'relative', top: '0'}"
```

Comment:
Given the additional styles, should we add a class for this?

Reply:
Created new class and moved style